samples/client.c

```diff
@@ -267,40 +257,6 @@ do_get_device_region_info(int sock, struct vfio_region_info *region_info,
     }
 }
 
-static void
-mmap_sparse_areas(int *fds, struct vfio_region_info *region_info,
```

Comment:
Why are you removing this? I think still mapping sparse areas is valid test

Reply:
`mmap_sparse_areas` was only ever called on the migration region so it's not applicable anymore - I'm not sure I understand what it's doing either so I don't know how I could adapt it to demonstrate another region?

Reply:
In server.c VFU_PCI_DEV_BAR1_REGION_IDX region also has sparse mmaps

Reply:
Why does it expect 2 FDs for VFU_PCI_DEV_BAR1_REGION_IDX? I get that there are two sparse areas but don't we have one file backing them?

Reply:
Expecting 2 FDs is correct, the server might use multiple files for each sparse region of a device region. The fact that it wasn't clear means we should have a comment explaining that, could you do this in a separate commit?

Reply:
If it expects two FDs then are we just supposed to give it the same one twice or something? That seems awkward

Reply:
I remember having a look into server.c implementation, what I got was it sends just 1 fd.